.gitignore

```diff
@@ -6,4 +6,15 @@ dockerfiles/run/**/keystore
 dockerfiles/run/**/glowroot
 .m2
 test-run.log
-build
\ No newline at end of file
+build
+
+.idea
+*.iml
+
+**/.settings/*.prefs
+**/.classpath
+**/.project
+derby.log
+.factorypath
+
+mailbox/var/
```

Comment:
We should differenciate files generated by the build or James that are legitimate to ignore here, and files generated by your local IDE that should be ignored within your local configuration (for example ~/.gitIgnore )

Reply:
Thanks. I'm not sure the `~/.gitignore` thing is good. 
I'll remove the entries for now. 
I'm not convinced this is the best approach.